### PR TITLE
Updating the tap-delay article for 2016

### PR DESCRIPTION
**Pull request by jakearchibald**
* Added info on iOS support
* Updated info on IE/Edge support
* Removed/condensed justification
* Removed appeal for feedback

@RByers - how's this?

### CODE REVIEW

File: src/content/en/updates/posts/2013/12/300ms-tap-delay-gone-away.markdown [hunk 1]
```diff
@@ -12,105 +12,46 @@ tags:
   - performance
   - touchevent
 ---
-<p>You'll find large articles throughout this site dedicated to shaving 10ms here and 90ms there in order to deliver a fast and fluid user experience. Unfortunately every touch-based mobile browser, across platforms, has an artificial ~300ms delay between you tapping a thing on the screen and the browser considering it a "click". When people think of the web as being sluggish compared to native apps on mobile, this is this one of the main contributors.</p>
 
-<p>However, as of <a href="https://play.google.com/store/apps/details?id=com.chrome.beta">Chrome 32 for Android</a>, which is currently in beta, this <strong>delay is gone</strong> for <strong>mobile-optimised</strong> sites, <strong>without removing pinch-zooming</strong>!</p>
+<style>
+.phone-screenshot {
```
Comment (jakearchibald) [inline src/content/en/updates/posts/2013/12/300ms-tap-delay-gone-away.markdown]: Is there a better way to do article specific styles?

Comment (jpmedley) [inline src/content/en/updates/posts/2013/12/300ms-tap-delay-gone-away.markdown]: In general, I don't know. There is a liquid macro that lets you specify width. 
Yeah, screenshot!{:width="360px"}

File: src/content/en/updates/posts/2013/12/300ms-tap-delay-gone-away.markdown [hunk 2]
```diff
@@ -12,105 +12,46 @@ tags:
...
-
-<p>We tried to imagine a user this may affect, someone who:</p>
-
-<ul>
-<li>has a motor impairment that prevents multi-touch interaction such as pinch-zoom, but not two taps in the same area within 300ms</li>
-<li>has a minor visual impairment that is overcome by the small amount of zooming provided by double-tap on mobile optimised sites</li>
-</ul>
-
-<p>But they're catered for by the text sizing tools in Chrome's settings, or the screen magnifier in Android, which covers all sites and native apps, and can be activated by triple-tap.</p>
+This sets the viewport width to the same as the device, and is general best-practice for mobile-optimised sites. With this tag, browsers assume you've made text readable on mobile, and the double-tap-to-zoom feature is dropped in favour of faster clicks.
```
Comment (jpmedley) [inline src/content/en/updates/posts/2013/12/300ms-tap-delay-gone-away.markdown]: 'and is general best-practice' > 'and is generally a best practice'. Please fix.

**Comment by jpmedley** [comment]
Jake, I found one sentence that needs fixed.

File: src/content/en/updates/posts/2013/12/300ms-tap-delay-gone-away.markdown (hunk 1 above)
Comment (petele) [inline src/content/en/updates/posts/2013/12/300ms-tap-delay-gone-away.markdown]: In the current system, no. If there's a style you want, we can look at adding it.

**Pull request merged**